src/ensembl/src/content/app/entity-viewer/gene-view/components/proteins-list/proteins-list-item-info/ProteinsListItemInfo.tsx

```diff
@@ -124,7 +120,7 @@ const ProteinsListItemInfo = (props: Props) => {
 
   useEffect(() => {
     const abortController = new AbortController();
-    if (xrefLoadingState === LoadingState.SUCCESS && !uniprotXref) {
+    if (summaryStatsLoadingState === LoadingState.LOADING && !uniprotXref) {
```

Comment:
It looks like the SummaryStats will never get fetched as this condition will always be true during the initial render. This will in turn set the `summaryStatsLoadingState` to SUCCESS, which would mean that the second `if` statement condition will never return true.

Reply:
> as this condition will always be true during the initial render

Why would it always be true during the initial render? `uniprotXref` comes from the transcript -> product_generating_context -> product -> external_references, which is already fetched and passed down to this component.

Try running this branch on your local machine. Does the problem appear there?

Reply:
>uniprotXref comes from the transcript -> product_generating_context -> product -> external_references, which is already fetched and passed down to this component.

No, it's not. 
It is only getting set here:
https://github.com/Ensembl/ensembl-client/pull/455/files#diff-5f1e049df55be7bf2c2bccf34cb18736d9f52f263aa58d02c44cce3bb417ab59R106

Reply:
> Try running this branch on your local machine. Does the problem appear there?

The problem mentioned in this PR doesn't seem to be occurring but I think the stats doesn't get displayed anymore.

Reply:
I've updated the PR:

- get uniprotXref directly from the transcript passed as a property; do not wait until the transcript is enriched with protein domains
- improve the coordination between the top and the bottom part of the component while loading (PDBe is responding faster than our rest today; so I was getting protein stats appearing before the domains have even been downloaded)